cranelift-codegen/meta/src/cdsl/ast.rs

```diff
@@ -11,7 +11,6 @@ use std::fmt;
 pub enum Expr {
     Var(VarIndex),
     Literal(Literal),
-    Apply(Apply),
```

Comment:
I don't really understand how this can go away?

Reply:
This was Rustc complaining that it was unused after I changed some pub to pub(crate). In fact, Apply expressions appear only in DummyExpr, which gets manipulated by xform.rs, but never rewritten to an Expr. Instead, it's embedded as part of a Def.